test/commands/commit.spec.js

```diff
@@ -142,6 +143,7 @@ describe('commit command', () => {
         getEmojis.mockResolvedValue(stubs.gitmojis)
         mockProcess.mockProcessExit()
         process.argv[3] = stubs.argv
+        execa.mockReturnValueOnce(Promise.resolve({ stdout: 'master' }))
```

Comment:
I think we should update the describe to match to the new spec

Reply:
like `while committing on master`?